src/acl/external/eDirectory_userip/ext_edirectory_userip_acl.cc

```diff
@@ -1211,10 +1102,9 @@ static int
 SearchIPLDAP(edui_ldap_t *l)
 {
     ber_len_t i, x;
-    ber_len_t j, k;
-    ber_len_t y, z;
-    int c;
-    char bufa[EDUI_MAXLEN], bufb[EDUI_MAXLEN], hexc[4];
+    ber_len_t j;
+    ber_len_t z;
+    char bufa[EDUI_MAXLEN], bufb[EDUI_MAXLEN];
```

Comment:
Since you are changing this, please use one declaration per statement/line.

Reply:
done